src/_data/whats-new.yml

```diff
@@ -4,8 +4,67 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Tue Jan 28 19:11:00 2020
+updated: Tue Feb 4 12:02:47 2020
 entries:
+- description: Updated the Magento CLI reference ([Commerce](https://devdocs.magento.com/guides/v2.2/reference/cli/magento-commerce.html)
+    | [Open Source](https://devdocs.magento.com/guides/v2.2/reference/cli/magento.html)).
```

Comment:
![Screen Shot 2020-02-04 at 12 58 36](https://user-images.githubusercontent.com/12731225/73777523-3bd49f00-474f-11ea-8eeb-799324aa106f.png)
looks like we cannot use a pipe in the description. This comes from my PR, I'll change it.

Reply:
done